src/concrete/ml/quantization/quantized_ops.py

```diff
@@ -953,7 +961,17 @@ def q_impl(
         # This is going to be compiled with a PBS (along with the following activation function)
         # Note that we don't re-quantize the output of the conv, this will be done by
         # any Gemm/Add/Conv layers that follow
-        m_matmul = q_input.quantizer.scale * q_weights.quantizer.scale
+        m_matmul: npt.NDArray[numpy.float64]
+        if q_weights.quantizer.scale.shape == tuple():
+            m_matmul = q_input.quantizer.scale * q_weights.quantizer.scale
+        else:
+            # TODO: add assert on shapes
+            weight_quant_scale = numpy.transpose(
+                q_weights.quantizer.scale,
+                axes=(1, 0, 2, 3),
```

Comment:
same

Reply:
same answer, the assert above (to be added) will catch any issue but it should always have this shape for conv2d.

I'll add some errors in the case of conv1d or conv3d